nemo/collections/asr/metrics/rnnt_wer.py

```diff
@@ -397,6 +403,44 @@ def decode_ids_to_tokens(self, tokens: List[int]) -> List[str]:
         """
         raise NotImplementedError()
 
+    @abstractmethod
```

Comment:
I'd rather not make it abstract, instead by default raising an error that it is unimplemented. 

Subclass must check if tokenizer supports it, then return something, else error out

Reply:
addressed